Resources/public/js/views/services/ez-navigationhubviewservice.js

```diff
@@ -23,6 +23,7 @@ YUI.add('ez-navigationhubviewservice', function (Y) {
         initializer: function () {
             this.on('*:logOut', this._logOut);
             this.after('*:navigateTo', this._navigateTo);
+            this.after('*:activeNavigationChange', this._activeNavigationChange);
```

Comment:
That used to be its name… then I followed the examples of the ones above, named after the even :( I'll fix it.